app/code/core/Mage/Eav/Model/Entity/Attribute/Source/Boolean.php

```diff
@@ -53,6 +53,9 @@ public function getAllOptions()
                 ),
             );
         }
+        if ($withEmpty) {
+            array_unshift($r, ['value' => '', 'label' => '']);
```

Comment:
@ADDISON74 error is correct, `$r` is undefined.

Reply:
My bad. Fixed it.